dask_expr/_reductions.py

```diff
@@ -240,6 +310,129 @@ def _tree_repr_lines(self, indent=0, recursive=True):
         return lines
 
 
+class ApplyConcatApply(Expr):
```

Comment:
Did anything change here or is it just at a different place?

Reply:
Ah, I move things around, because I "needed" to at one point. Sorry about that. I don't need this re-ordering anymore, so I might try to move it back to reduce the diff.